src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -20,56 +20,51 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import kubernetesClusterExplorer from 'images/kubernetes_screenshot-full_cluster-explorer.gif'
+import kubernetesClusterExplorer from 'images/kubernetes_screenshot-crop_cluster-explorer.webp'
 
 import kubernetesDashboard from 'images/kubernetes_screenshot-full_dashboard.webp'
 
 import kubernetesEvents from 'images/kubernetes_screenshot-full_events.gif'
 
-Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
 
-Thanks to our [cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer), you can cut through layers of complexity to see how your cluster is performing, from the heights of the control plane down to applications running on a single pod.
+Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you a quick and easy way to see what's going on with your Kubernetes clusters and workloads, whether they're hosted on-premises or in the cloud.
 
-<img
-title="New Relic - Kubernetes Cluster Explorer"
-alt="New Relic - Kubernetes Cluster Explorer"
-src={kubernetesClusterExplorer}
-/>
+Once you've got the [Kubernetes integration up and running](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/), you can start monitoring all the services running on Kubernetes in one place with to our [cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer). It'll show you everything from the heights of the control plane down to applications running on a single pod, so you can get to the heart of how your cluster is performing.
```

Comment:
Should we use a more generic term to refer to our K8s UIs?
Instead of "cluster explorer" I would recommend something like "Kubernetes curated experiences" or "Kubernetes out-of-the-box experiences"

The page linked not only refer to the K8s navigator but to other UIs.

Remember that the cluster explorer refers to the old UI we want to deprecate soon (the one represented in a circular way)